pandas/core/common.py

```diff
@@ -1420,350 +1420,6 @@ def _fill_zeros(result, x, y, name, fill):
     return result
 
 
-def _interp_wrapper(f, wrap_dtype, na_override=None):
```

Comment:
you *might* be able to strip some imports here, you might need a linter to find these though